README.md

```diff
@@ -1,77 +1,107 @@
 # ForageSDK
 
-The ForageSDK processes Electronic Benefits Transfer (EBT) payments in your e-commerce application. It provides secure user interfaces for collecting and tokenizing an EBT cardholder's PAN and accepting an EBT cardholder's PIN to execute a balance check and process a payment.
+The ForageSDK is a Swift library for securely processing EBT payments. You can use the library to add all of the essential EBT checkout operations to your iOS app:
```

Comment:
This is a nit, and I need to change the language on the Forage JS intro page to match as well: FNS no longer requires balance checks. In fact, if you have guest checkout on your website, then FNS prohibits you from offering a balance check [0]. 

I'd recommend scrapping "essential" in favor of something like "the following", so it couldn't be construed that balance check is required. 

[0] 
- [Slack thread](https://joinforage.slack.com/archives/C04KZ0F9P60/p1697826792476429) where I learned this 
- Linear ticket where I've been tracking [outstanding updates](https://linear.app/joinforage/issue/TW-337/update-other-reference-docs-to-note-fns-fraud-guest-checkout)

Reply:
Nice catch! Will update